video_processing/script.py

```diff
@@ -70,64 +90,86 @@ def get_heart_rate(file_url):
         h, s, v = cv2.split(im)
         full_video.append(int(np.sum(v)))
 
-    print("full_video")
-    print(len(full_video))
-
-    values = full_video
+    print('full video len: {}'.format(len(full_video)))
 
-    maxi = np.max(values)
-    mini = np.min(values)
+    max_y = np.max(full_video)
+    min_y = np.min(full_video)
 
-    dist = maxi - mini
+    dist = max_y - min_y
 
     # TODO: tady bude mozna treba to rozseparovat do mensich casti a min/max urcit zvlast
-    peaks, properties = scipy.signal.find_peaks(values, prominence=dist / 6)
-    x_axis_in_sec = [i / fps for i in range(len(values))]
+    peaks, properties = scipy.signal.find_peaks(full_video, prominence=dist / 6)
+    x_axis_in_sec = [i / fps for i in range(len(full_video))]
     peaks_in_sec = [i / fps for i in peaks]
     peaks_distances = []
     for previous, current in zip(peaks_in_sec, peaks_in_sec[1:]):
         peaks_distances.append(current - previous)
 
-    # values
-    pulse_wave = [x_axis_in_sec, values]
-    plt.plot(x_axis_in_sec, values)
+    ppg = [x_axis_in_sec, full_video]
+    plt.plot(x_axis_in_sec, full_video)
 
     # peaks
-    peaks_arr = [peaks_in_sec, [int(values[i]) for i in peaks]]
-    plt.plot(peaks_in_sec, [int(values[i]) for i in peaks], 'ro')
-    plt.axhline(max(values))
-    plt.axhline(min(values))
+    peaks_arr = [peaks_in_sec, [int(full_video[i]) for i in peaks]]
+    plt.plot(peaks_in_sec, [int(full_video[i]) for i in peaks], 'ro')
+    plt.axhline(max(full_video))
+    plt.axhline(min(full_video))
 
     plt.xlabel('time(s)')
     plt.show()
 
     # BPM
     bpm = len(peaks)
-    bpm *= round(60/video_duration)
-    print("bpm")
-    print(bpm)
+    bpm *= round(60 / video_duration)
+    print('bpm: {}'.format(bpm))
+
+    # RR intervals
+    rr_tachogram_plot(peaks_distances)
+    poincare_plot(peaks_distances)
 
-    if bpm <= 50:
+    os.remove(path)
+
+    if bpm <= 40:
         return make_response(jsonify({
             'error': 'Invalid input',
-            'message': 'Poor video quality.'
+            'message': 'Poor video quality, please repeat the measurement.'
         }), 400
         )
-
-    # RR intervals
-    plt.figure(2)
-    plt.plot(peaks_distances, 'bo')
-    ax = plt.gca()
-    ax.set_ylim([0, 1])
-    plt.show()
-
-    os.remove(path)
+    elif bpm < 60:
+        return make_response(jsonify({
+            'conclusion': 'Bradycardia',
+            'message': 'A normal resting heart rate for adults ranges from 60 to 100 beats per minute (bpm).' +
```

Comment:
Warning! Your heart rate is {}. 
'A normal resting heart rate for adults ranges from 60 to 100 beats per minute (bpm).' + A heart rate of {} is thus considered as bradycardia.
                       'If you\'re not an athlete and your bpm is lower than 60 bpm, it might be a sign of bradycardia, which could be an indicator of other health conditions. ' +
Please, double check your heart rate (repeat the measurement, use different ways of measuring heart rate). If you do not feel well, call a doctor.